### PR TITLE
Adding trending page

### PR DESCRIPTION
**Pull request by b0llu**
## What to check 
- [ ] src/Pages/TrendingPage/TrendingPage.jsx

## What not to check
- [ ] Do Not Check Any Other Files

### CODE REVIEW

File: src/Pages/TrendingPage/TrendingPage.jsx [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    <section>
+      <div className="trending-header">
+        <h1>Trending Today</h1>
+      </div>
+      {loading ? (
+        <Loader />
+      ) : (
+        <div className="trending-video-container">
+          {videos
+            .filter((video) => (video.trending ? video : null))
```
Comment (nayyyhaa) [inline src/Pages/TrendingPage/TrendingPage.jsx]: In the filter function you can just give a condition.. it'll automatically return the looped element if it's true, else null.
like: `videos.filter(video => video.trending)`

**Review by nayyyhaa — APPROVED** [review]
just one suggestion mentioned. rest your code looks good. you can merge it. 👍

File: src/Pages/TrendingPage/TrendingPage.jsx (hunk 1 above)
Comment (b0llu) [reply src/Pages/TrendingPage/TrendingPage.jsx]: thanks for pointing it out i will change it

**Review by Dev-AkshatGupta — APPROVED** [review]
Very small PR well maintained nothing to suggest. You can merge this

**Pull request merged**